crates/net/types/Cargo.toml

```diff
@@ -6,7 +6,7 @@ rust-version.workspace = true
 license.workspace = true
 homepage.workspace = true
 repository.workspace = true
-description = "Network types and utils"
+description = "Peer related types"
```

Comment:
is this correct?
https://github.com/paradigmxyz/reth/blob/fe1b962003c08ee8d8db2e8efbb7686c9ed4c568/crates/net/types/src/lib.rs#L1

Reply:
my bad, committed by accident hehe